contribute.md

```diff
@@ -0,0 +1,9 @@
+# Contribute to This Document
+
+OlympusDAO operates autonomously, so everyone is welcome to contribute to this GitBook guide. This page outlines the contribution workflow.
+
+## Join Our Community
+
+We have a [DAO Discord channel](https://discord.com/channels/838651642190495804/838655387640004648) where we discuss on anything related to the development of the GitBook documentation. Join our community there and have a meaningful discussion with other members to further improve the content quality.
```

Comment:
Is there something specific about the onboarding process people have to follow? I can see that people just joining don't see anything in the first place. Right now you have to fill that google doc or something and wait. And this is probably also subject to change.

Reply:
```suggestion
Join our [DAO Discord](https://discord.gg/42xFV68uEf) and make sure you fill up the onboarding form there. Signal your interest in content creation and you will be assigned a corresponding role. After that, you can collaborate with other members in the Content team to further improve this document.

*Note: The onboarding process might be subject to change in the future. Worry not, just ask around in the discord and our helpful community will guide you through the onboarding process.*
```

Reply:
Good point. How about this change? Added a note that the onboarding process is subject to change

Reply:
Let's go with this.